CHANGES

```diff
@@ -87,6 +87,7 @@ Bugs fixed
   language and ``'xelatex'`` or ``'lualatex'`` as :confval:`latex_engine`
   (refs: #5251)
 * #5248: LaTeX: Greek letters in section titles disappear from PDF bookmarks
+* LaTeX: document how to allow use of Unicode literals in math (refs: #5249)
```

Comment:
I think this isn't a bug fix. So I feel a bit strange for this entry.

Reply:
of course, don't know why I added this entry. I will remove that. And I will try to make the added explanations a bit less verbose. (as matter of fact I find our LaTeX doc occupies already much space...)